README.md

```diff
@@ -80,10 +80,10 @@ See the [LICENSE file](LICENSE.txt) for license rights and limitations.
 
 ## Get the latest news
 
-- **Twitter** - Follow [Mattermost](https://twitter.com/mattermost).
+- **Twitter** - Follow [Mattermost on X, formerly Twitter](https://twitter.com/mattermost).
```

Comment:
Since the name has changed, I recommend styling this line as:

- **X** - Follow [Mattermost on X, formerly Twitter](https://twitter.com/mattermost).

Reply:
cc https://github.com/mattermost/mattermost/pull/25205 makes the same change